app/src/main/java/org/sopt/official/feature/auth/utils/GoogleLoginManager.kt

```diff
@@ -0,0 +1,45 @@
+package org.sopt.official.feature.auth.utils
+
+import android.content.Context
+import androidx.credentials.CredentialManager
+import androidx.credentials.CustomCredential
+import androidx.credentials.GetCredentialRequest
+import com.google.android.libraries.identity.googleid.GetGoogleIdOption
+import com.google.android.libraries.identity.googleid.GoogleIdTokenCredential
+import org.sopt.official.BuildConfig.serverClientId
+import org.sopt.official.common.coroutines.suspendRunCatching
+import timber.log.Timber
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class GoogleLoginManager @Inject constructor() {
+    suspend fun getGoogleIdToken(context: Context): String {
```

Comment:
```suggestion
class GoogleLoginManager @Inject constructor(
    @ApplicationContext private val context: Context
) {
    suspend fun getGoogleIdToken(context: Context): String {
```

이런식으로 넣으면 어떨까요? 혹시 Activity Context 필요하면 @ActivityComponent에서 주입하고 @ActivityContext 사용하는 방법도 있습니다!